docs/user-guide/query/gaffer-syntax/filtering.md

```diff
@@ -654,6 +654,347 @@ total for all the `added` and `removed` properties.
+                                "class" : "uk.gov.gchq.koryphe.impl.binaryoperator.Min"
+                                }
+                            } ]
+                        }
+                    }
+                }
+            }
+        }
+        ```
+
+    === "Python"
+
+        ```python
+         elements = g_connector.execute_operation(
+            operation = g.GetElements(
+                input = [g.EntitySeed(vertex = "John")]
+                view = g.View(
+                    edges = [
+                        g.ElementDefinition(
+                            group = 'Commit',
+                            group_by = [],
+                            pre_aggregation_filter_functions=[
+                                g.PredicateContext(
+                                    predicate=g.Or(
+                                        predicates=[
+                                            g.InDateRange(
+                                                start="2024/05/01",
+                                                end="2024/05/05",
+                                                start_inclusive=True,
+                                                end_inclusive=False
+                                            )
+                                        ]
+                                    )
+                                )
+                            ],
+                            aggregate_functions = [
+                                g.BinaryOperatorContext(
+                                    selection=["added"],
+                                    binary_operator = g.Min()
+                                )
+                            ]
+                        )
+                    ]
+                )
+            )
+        )
+        ```
+
+More commonly, you will want to set the groupBy to empty for all groups in your schema.
+This can be done using the ``globalElements` section in the view:
+
+!!! example ""
+
+    === "Java"
+
+        ```java
+        final GetElements getEdgesSummarised = new GetElements.Builder()
+            .input(new EntitySeed("John"))
+            .view(new View.Builder()
+                    .globalElements(new GlobalViewElementDefinition.Builder()
+                            .groupBy()
+                            .build())
+                    .build())
+            .build();
+        ```
+
+    === "JSON"
+
+        ```json
+        {
+            "class": "GetElements",
+            "input": [
+                {
+                    "class": "EntitySeed",
+                    "vertex": "John"
+                }
+            ],
+            "view": {
+                "globalElements": {
+                    "groupBy" : []
+                }
+            }
+        }
+        ```
+    === "Python"
+
+        ```python
+        elements = g_connector.execute_operation(
+            operation = g.GetElements(
+                input = [g.EntitySeed(vertex = "John")]
```

Comment:
I think on most of the python examples you are missing the `,` between arguments e.g.
```python
input = [g.EntitySeed(vertex = "John")],
view = ...
```